src/dotnet/Program.cs

```diff
@@ -121,22 +121,47 @@ private static int ProcessArgs(string[] args)
                 ["test"] = TestCommand.Run
             };
 
+            int exitCode = 100;
+            
+            string arguments = string.Empty;
+
+
             Func<string[], int> builtIn;
             if (builtIns.TryGetValue(command, out builtIn))
             {
-                return builtIn(appArgs.ToArray());
+                exitCode = builtIn(appArgs.ToArray());
+                
+                appArgs.ToList().ForEach(a => { arguments += a + " "; });
+
+            }
+            else
+            {
+                CommandResult result = Command.Create("dotnet-" + command, appArgs, FrameworkConstants.CommonFrameworks.NetStandardApp15)
```

Comment:
According to # 10 in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md this shouldn't be `var`.

Since cli is under 'dotnet', I think it makes sense to follow these, that way we have a consistent set of coding-style guidelines across all 'dotnet' repos.